tests/drivers/spi/spi_loopback/boards/nucleo_h743zi.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_1"
+
+CONFIG_SPI_STM32_DMA=n
+CONFIG_SPI_STM32_INTERRUPT=y
+
+CONFIG_SPI_ASYNC=y
+
+CONFIG_SPI_LOOPBACK_MODE_LOOP=n
+
+CONFIG_NOCACHE_MEMORY=y
```

Comment:
Is this actually required if DMA is not used?

Reply:
yes, not needed here